src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -286,6 +286,20 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
       )
   }
 
+  lazy private[overwatch] val notebookCommandsModule = Module(3018, "Gold_NotebookCommands", this, Array(1004,3004,3005))
+  lazy private val appendNotebookCommandsProcess: () => ETLDefinition = {
+    () =>
+      ETLDefinition(
+        BronzeTargets.auditLogsTarget.asIncrementalDF(notebookCommandsModule, BronzeTargets.auditLogsTarget.incrementalColumns),
```

Comment:
need to add 1 day lag for commands that arrive late into the audit bronze source.

Reply:
Agree. Added